File: patient.js
import { LightningElement, track } from 'lwc';

export default class Patient extends LightningElement {
    @track name = 'John Doe';
    @track dob = '1970-01-01';
    @track address = '123 Main St';

    save() {
        // Save the patient information
    }
}

File: patient.rs
use serde::{Deserialize, Serialize};
use validator::Validate;

#[derive(Clone, Debug, Default, Serialize, Deserialize, Validate)]
pub struct Patient {
    #[validate(length(min = 1))]
    pub name: String,
    #[validate(length(min = 1))]
    pub dob: String,
    #[validate(length(min = 1))]
    pub address: String,
}
